webcompat/issues.py

```diff
@@ -26,10 +50,12 @@ def report_issue(form, proxy=False):
     submit_type = form.get('submit_type')
     if proxy and submit_type == 'github-proxy-report':
         # Return a Response object.
+        path = 'repos/{0}'.format(PRIVATE_REPO_URI)
         response = proxy_request('post',
                                  path,
-                                 data=json.dumps(build_formdata(form)))
+                                 data=json.dumps(unmoderated_issue()))
         json_response = response.json()
+        report_private_issue(form, json_response.get("html_url"))
```

Comment:
This is slower than before, because we have to wait for 2 GitHub responses. I've filed https://github.com/webcompat/webcompat.com/issues/3148 to improve this, but I don't think we need to block anon reporting on it.

Reply:
So there's something I have difficulties to understand here. Why `report_private_issue`. `proxy_request` is already here. it's just the parameters which are changing.

I mean the idea to have function names for each tasks could clearly identified each scenario, but in this case, we probably need to give a name for each of them. Here it seems suddenly unbalanced.

Reply:
No specific reason, I'll add a `report_public_issue` method to restore balance to the force.